tests/testproject/TestProject.IntegrationServiceA/SqlServer/SqlServerExtensions.cs

```diff
@@ -18,7 +18,7 @@ private static async Task<IResult> VerifySqlServerAsync(SqlConnection connection
             var policy = Policy
                 .Handle<SqlException>()
                 // retry 60 times with a 1 second delay between retries
-                .WaitAndRetryAsync(60, retryAttempt => TimeSpan.FromSeconds(1));
+                .WaitAndRetryAsync(10, retryAttempt => TimeSpan.FromSeconds(1));
```

Comment:
@eerhardt @sebastienros Is there a better way to express a "overall timeout" for the attempts here, so it all completes under n mins? Otherwise, this keeps retrying and the caller (test) fails because of it's own timeout (2mins).